commands.js

```diff
@@ -701,6 +701,9 @@ var commands = exports.commands = {
 			}
 			return connection.sendTo(target, "|noinit|joinfailed|You do not have permission to join '" + target + "'.");
 		}
+		if (toId(target) in Rooms.aliases) {
+			connection.send(">" + toId(target) + "\n|deinit");
+		}
```

Comment:
Doesn't `|noinit` send to the alias as well as the room name? Or am I remembering things wrong

Either way I'll do it when I get home